src/NewTodo.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class NewTodo extends React.PureComponent {
+  state = {
+    users: this.props.user,
+    userID: 0,
+    taskDescription: '',
+    taskValidation: 1,
+    userValidation: 1,
+    value: '',
+  }
+
+  createList = arr => (
+    arr.map(user => (
+      <option key={user.email}>{user.name}</option>
+    ))
+  )
+
+  selectedUser = (event) => {
+    const i = this.state.users.find(el => el.name === event.target.value);
+
+    this.setState(({ userID, userValidation, value }) => ({
+      userID: i.id,
+      userValidation: 1,
+      value: i.name,
+    }));
+  }
+
+  taskTitle = (event) => {
+    this.setState({ taskDescription: event.target.value });
+    this.setState({ taskValidation: 1 });
+  }
+
+  selectedPersonValidation = () => (
+    this.state.userValidation !== 0
+      ? 'form__validation_user'
+      : 'form__validation_user_alert'
+  )
+
+  newTaskValidation = () => (
+    this.state.taskValidation !== 0
+      ? 'form__validation_text'
+      : 'form__validation_text_alert'
+  )
```

Comment:
all those can be replaced with methods from classnames lib

Reply:
fixed